outboarding_guide_outline.md

```diff
@@ -0,0 +1,109 @@
+# Onboarding Guide Outline
+
+## Overview
+
+Each major language that Tilt supports should have an onboarding guide that demonstrates:
+- The simplest way to set up Tilt with that language
+- The recommended way to set up Tilt with that language ("the fastest deployment time")
+- Walkthrough a clear path between those two setups
+
+This guide should have working code samples that the user can check out and explore themselves.
+
+We're basically cannibalizing the narrative of "The Quest for the Fastest Deployment Time"
+
+Slides: https://docs.google.com/presentation/d/1Vs4a4SSI4BE2094bloBY410FJkaH1WucHvVXcOJ78H8/edit#slide=id.g76703e551a_2_427
+Code: https://github.com/windmilleng/fast
+
+But with a couple changes in scope:
+
+- More detailed explanation of what the Tiltfile does
+- Fewer intermediate experiments
+- A simpler app
+
+## Code Structure
+
+All repos should borrow the code structure from https://github.com/windmilleng/fast
+
+The repo name should be `windmilleng/tilt-example-LANGUAGE`. So Go would be
+`tilt-example-go`, Java would be `tilt-example-java`, etc.
+
+Each subdirectory would be a version of the app:
+
+- 0-base: The simplest possible app
+- 1-measured: The simplest possible app, but modified to measure its own deployment time.
+- 2-name-a: First optimization
+- 3-name-b: And so on
+- 4-recommended: The recommended version of the app.
+
+## Narrative Structure
+
+The best indicator of a healthy development workflow is a short feedback loop.
+
+Kubernetes is a huge wrench in the works.
+
+Let’s fix this.
```

Comment:
We might want to add a link at the top that says, "if you just want the final version that is ready to use, click here to skip the exploration."